test/Models/RegionProviderSpec.js

```diff
@@ -26,9 +25,8 @@ beforeEach(function() {
 
 describe('RegionProvider', function() {
   it('parses WFS xml correctly', function(done) {
-    loadText('test/csv/mini_ced.xml').then(function(xml) {
-        ced.loadRegionsFromXML(xml);
-        expect(ced.regions.length).toEqual(6);
+    ced.loadRegionIDs().then(function(json) {
```

Comment:
btw @kring's doing some cool work on this right now using `sinon.fakeServer.create()`, to solve the problem of failing unit tests on IE9 (which cannot load data from the region mapping server without a proxy).